reference/forms/types/repeated.rst

```diff
@@ -177,6 +178,21 @@ type
 The two underlying fields will be of this field type. For example, passing
 ``PasswordType::class`` will render two password fields.
 
+compare
+~~~~~~~
+
+**compare**: ``callable`` **default**: ``null``
+
+The two underlying fields' values will be compared by this callable. The callable receives two values to compare, and should return a boolean: true if the fields are considered equal, false otherwise.
```

Comment:
I think you can omit the first sentence and start directly with `The callable receives two values to compare`.